tests/README.md

```diff
@@ -33,7 +33,7 @@ In order to make these images available for remote testing you can use the follo
 
 ```
 > docker tag jetty:9.4 gcr.io/{project}/jetty:9.4
-> gcloud docker push gcr.io/{project}/jetty:9.4 
+> gcloud docker -- push gcr.io/{project}/jetty:9.4 
```

Comment:
The latest version of gcloud warn that the -- argument should be used to separate the gcloud arguments from the docker arguments.  Not using -- is deprecated and eventually will not be supported.